-- Restrict the CMP0026 OLD policy to Apple platforms.  CMake 4.0 no
-- longer permits OLD behavior for this policy, and it is only needed for
-- building debug symbol bundles on Apple platforms.
--- CMakeLists.txt.orig	2024-04-05 00:00:00 UTC
+++ CMakeLists.txt
@@ -38,7 +38,7 @@
 # Allow reading the LOCATION property of a target to determine the eventual
 # location of build targets. This is needed when building the debugging symbols
 # bundles for Apple platforms.
-if (POLICY CMP0026)
+if (APPLE AND POLICY CMP0026)
   cmake_policy(SET CMP0026 OLD)
 endif()
